apps/music/components/gaia-fast-list/gaia-fast-list.js

```diff
@@ -356,14 +382,24 @@ var GaiaFastListProto = {
         text-overflow: ellipsis;
       }
 
-      ::content .image ~ h3,
-      ::content .image ~ p {
-        padding-inline-end: 60px;
+      ::content :-moz-dir(ltr) .image ~ h3,
+      ::content :-moz-dir(ltr) .image ~ p {
+        padding-right: 52px;
+      }
+
+      ::content :-moz-dir(rtl) .image ~ h3,
+      ::content :-moz-dir(rtl) .image ~ p {
+        padding-left: 52px;
```

Comment:
That may have been because of nested Shadow DOM. But, in this patch, I believe the search results are no longer in Shadow DOM.